s3_deploy.sh

```diff
@@ -2,7 +2,7 @@
 SRC_DIR='dist'
 DISTRIBUTION_ID='E1YPVV3YLYS4J7'
 # name of branch to deploy to root of site
-PRODUCTION_BRANCH='production'
+PRODUCTION_BRANCH='master'
```

Comment:
I'm not convinced this is the best starting point for a new project. Most of the projects I've worked on lately, including CLUE, merge to `master` upon completion of code review and then deploy to `production` after QA and go-ahead from stakeholders. If `master` gets pushed to the root of the site automatically then a different process is required. This seems to invite inadvertent updating of the production site. I'd rather have the default starting point be the safer/more conservative approach and allow individual projects to change it where it makes sense to do so.

Reply:
Okay, I agree on that. I guess there's no right answer here. Depends whether the project uses root directory as the main deployment or tagged versions. I've been working mostly on projects that use tagged version and where the root dir isn't used much. But deploying `production` is more conservative, I agree, so let's leave it like that. Just fixed that, thanks!